ivy/functional/ivy/data_type.py

```diff
@@ -1617,18 +1629,29 @@ def function_supported_dtypes(fn: Callable, recurse: bool = True) -> Tuple:
             "in a particular backend"
         ),
     )
-    supported_dtypes = set(_get_dtypes(fn, complement=False))
-    if recurse:
-        supported_dtypes = _nested_get(
-            fn, supported_dtypes, set.intersection, function_supported_dtypes
-        )
-
-    return tuple(supported_dtypes)
+    if hasattr(fn, "partial_mixed_handler"):
```

Comment:
Would be great to update the return signature of this function in it's docstring, mentioning that it will be a dict for partial mixed functions

Reply:
Updated.